pkg/cloud/services/loadbalancer/loadbalancer.go

```diff
@@ -48,7 +48,7 @@ const loadBalancerProvisioningStatusActive = "ACTIVE"
 
 func (s *Service) ReconcileLoadBalancer(openStackCluster *infrav1.OpenStackCluster, clusterName string, apiServerPort int) (bool, error) {
 	loadBalancerName := getLoadBalancerName(clusterName)
-	s.scope.Logger().Info("Reconciling load balancer", "name", loadBalancerName)
+	s.scope.Logger().Info("Reconciling load balancer", "loadBalancerName", loadBalancerName)
```

Comment:
Lets hold off on this change until we have agreement in https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/1617#discussion_r1271990511

I suspect that we may decide we're going to stick with `name` here.